### ISSUE TITLE
Editing project tweaks

### ISSUE THREAD

**Issue by mkoo** [AmphibiaWeb/amphibian-disease-tracker#58]
- gray out ark since that is not editable
- calendar input for date fields (enforce ISO-date?)

**Comment by tigerhawkvok** [comment]
Dates aren't editable either -- they're derived from data.

Should they be editable in the case of no data?

**Comment by mkoo** [comment]
Yes, I'm talking about the Project date which does not need to come from
uploaded datasets.
